### PR TITLE
rtty: update to 8.1.0

### PR DESCRIPTION
**Pull request by JiaY-shi**
Maintainer: @zhaojh329 

Signed-off-by: Shi JiaYang [shi05275@163.com](mailto:shi05275@163.com)

### CODE REVIEW

File: utils/rtty/Makefile [hunk 1]
```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=rtty
-PKG_VERSION:=8.0.1
+PKG_VERSION:=8.1.0
 PKG_RELEASE:=2
```
Comment (zhaojh329) [inline utils/rtty/Makefile:12]: The `PKG_RELEASE` should be reset to 1.

Comment (JiaY-shi) [reply utils/rtty/Makefile]: > The `PKG_RELEASE` should be reset to 1.

done

**Comment by BKPepe** [comment]
Any indication of compile and run testing except the CI?

**Comment by JiaY-shi** [comment]
> Any indication of compile and run testing except the CI? Because the CI does not test this package.
> 
> ```
> Testing package libev in version 4.33 from libs/libev
> No test.sh script available
> 
> Testing package rtty-mbedtls in version 8.1.0 from utils/rtty
> No test.sh script available
> 
> Testing package rtty-nossl in version 8.1.0 from utils/rtty
> No test.sh script available
> 
> Testing package rtty-openssl in version 8.1.0 from utils/rtty
> No test.sh script available
> 
> Testing package rtty-wolfssl in version 8.1.0 from utils/rtty
> No test.sh script available
> ```

@BKPepe Sorry for the late reply, tested on gl-axt1800

**Pull request merged**